client/src/app/components/Header/header.css

```diff
@@ -0,0 +1,122 @@
+h1 {
+  font-size: 20px;
+  margin: 10px;
+  color: #312a31;
+}
+
+.header {
+  width: 100%;
+  margin: 0;
+  padding: 5px;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  border-bottom: 1px solid #e8e9eb;
+  box-shadow: 0 2px 2px 2px #e8e9eb;
+  font-size: 16px;
+}
+
+.logo {
+  font-size: 25px;
+  color: #ec5252;
+}
+
+.nav-container {
+  display: flex;
+  align-items: center;
+}
+
+ul {
```

Comment:
This is bad. It will target all the ul with global style. 
Same for the selector li bellow. 

CSS should be scoped to specific elements, to avoid pollution coming from everywhere.

Having a global CSS targeting all the app is OK, but only if we're clear about it. For instance by importing a file called global.css inside App.js